stats/test/handler.test.js

```diff
@@ -237,12 +239,43 @@ describe('HTTP request handler', () => {
         }
       )
       await assertResponseStatus(res, 200)
-      const stats = await res.json()
+      let stats = await res.json()
       assert.deepStrictEqual(stats, [
         { day: '2024-01-20', success_rate: 0.1, successful: '1', total: '10', successful_http: '0', success_rate_http: 0 },
         { day: '2024-01-21', success_rate: 0.1, successful: '1', total: '10', successful_http: null, success_rate_http: null },
         { day: '2024-01-22', success_rate: 0.1, successful: '1', total: '10', successful_http: null, success_rate_http: null }
       ])
+
+      res = await fetch(
+        new URL(
+          '/miners/retrieval-success-rate/summary?from=2024-01-20&to=2024-01-22',
```

Comment:
This test case (`it('handles successful_http values 0, null, undefined',`) is nested inside the test suite `describe('GET /retrieval-success-rate',`. It should test the endpoint `GET /retrieval-success-rate` only.

Please move this new block testing `/miners/retrieval-success-rate/summary` to a new test case inside the test suite `describe('GET /miners/retrieval-success-rate/summary'` later in this file.

https://github.com/filecoin-station/spark-stats/blob/f4918ee55e46138df3824a250d3a04ae100e6b2b/stats/test/handler.test.js#L463